app/src/main/java/com/example/rocketreserver/LaunchListFragment.kt

```diff
@@ -47,7 +45,7 @@ class LaunchListFragment : Fragment() {
             var cursor: String? = null
             for (item in channel) {
                 val response = try {
-                    apolloClient(requireContext()).query(LaunchListQuery(cursor = Input.fromNullable(cursor))).await()
+                    apolloClient(requireContext()).query(LaunchListQuery(cursor))
```

Comment:
Random thoughts: there's a question about whether this should be the default. It's more concise in 95% of the cases but there might be 5% out there that might require `Optional`